### ISSUE TITLE
input type="range" value binding works wrong

### ISSUE THREAD

**Issue by Oblomoff** [dotnet/aspnetcore#33499]
### input type="range" value binding works wrong
The following code renders input with actual value="100" instead of expected value="180" 
```Razor
<input type="range" value="@(180)" min="20" max="220" />
```

### To Reproduce

Create new Blazor Server project 
```PowerShell
dotnet new blazorserver
```
Add the code to any razor page
```Razor
<input type="range" value="@(180)" min="20" max="220" />
```
See also the attached project: [RangeBug.zip](https://github.com/dotnet/aspnetcore/files/6641892/RangeBug.zip)

### Further technical details
- ASP.NET Core version: 5.0.7
- dotnet --info
```
  .NET SDK (reflecting any global.json):
 Version:   5.0.301
 Commit:    ef17233f86

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19043
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.301\

Host (useful for support):
  Version: 5.0.7
  Commit:  556582d964

.NET SDKs installed:
  5.0.201 [C:\Program Files\dotnet\sdk]
  5.0.301 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.16 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.16 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.16 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```
- The IDE
```
Visual Studio Code
Version: 1.57.0 (user setup)
Commit: b4c1bd0a9b03c749ea011b06c6d2676c8091a70c
Date: 2021-06-09T17:18:42.354Z
Electron: 12.0.9
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19043
```

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us. Can you try using the same in html page (not Blazor) and see whether that will result in the same outcome?

**Comment by mkArtakMSFT** [comment]
We were able to confirm that this is a bug and we will see whether we can address this in the near future.

**Comment by SteveSandersonMS** [comment]
BTW as a workaround for now, try putting your `value` attribute after the `min`/`max` ones:

```razor
<input type="range" min="20" max="220" value="@(180)" />
```

This should avoid the issue until we're able to fix the underlying cause.

**Comment by Oblomoff** [comment]
@SteveSandersonMS Thank you, guys. This workaround worked for me. And why didn't I figure it out myself?

**Comment by MackinnonBuck** [comment]
Closing because this has been fixed in  #33831.
